docs/deployment/README.md

```diff
@@ -266,40 +222,77 @@ dev:                            # deployment name
   cmr:
     username: '{{CMR_USERNAME}}'
     password: '{{CMR_PASSWORD}}'
-    clientId: '<replace-with-daac-name>-{{stackName}}'
-    provider: CUMULUS
+    clientId: '<replace-with-daac-name>-{{stackName}}' # Client-ID submitted to CMR to identify origin of requests.
+    provider: CUMULUS                                  # Target provider in CMR
+
+  es:                               # Optional. Set to 'null' to disable elasticsearch.
+    name: myES5Domain               # Optional. Defaults to 'es5vpc'.
+    elasticSearchMapping: 2         # Optional, triggers elasticSearch re-bootstrap.
+                                    # Useful when e.g. mappings are updated.
+
+  params:                           # Optional override.
+    app:                            # Used for app stack when 'npm run deploy-app' or 'kes cf deploy --kes-folder app --template ./node_modules/@cumulus/deployment/app [...]' is used.
```

Comment:
Will Kes merge these params with the default ones from the deployment package or override them? I assume it merges them, just making sure

Reply:
Also, these inline comments seem a bit long/verbose. Could we just say "Optional additional parameters to pass to the app stack deployment"? Or if we want to keep these comments as is, maybe they could be a note below the code block?

Reply:
I believe it only overrides in the case of arrays. Pushing up an update.